packages/pilot/src/components/MetricList/List.js

```diff
@@ -0,0 +1,35 @@
+import React, { Fragment } from 'react'
+import PropTypes from 'prop-types'
+
+import ListItem from './ListItem'
+
+import styles from './style.css'
+
+const List = ({
+  items,
+}) => (
+  <Fragment>
+    <div className={styles.list}>
+      {items.map(({
+        icon,
+        title: itemTitle,
+        value,
+      }, index) => (
+        <ListItem
+          key={`${index}-${value}`} // eslint-disable-line react/no-array-index-key
```

Comment:
Só pra não passar dos 80 caracteres 😄 
```suggestion
          // eslint-disable-next-line react/no-array-index-key
          key={`${index}-${value}`}
```